bin/env.sh

```diff
@@ -1,28 +1,5 @@
 #!/bin/bash
 
-export volume=nexus3.edgexfoundry.org:10004/docker-edgex-volume:master
-export consul=nexus3.edgexfoundry.org:10004/docker-edgex-consul:master
-export configSeed=nexus3.edgexfoundry.org:10004/docker-core-config-seed-go:master
-export mongo=nexus3.edgexfoundry.org:10004/docker-edgex-mongo:master
-
-export coreData=nexus3.edgexfoundry.org:10004/docker-core-data-go:master
-export coreMetadata=nexus3.edgexfoundry.org:10004/docker-core-metadata-go:master
-export coreCommand=nexus3.edgexfoundry.org:10004/docker-core-command-go:master
-
-export supportLogging=nexus3.edgexfoundry.org:10004/docker-support-logging-go:master
-export supportNotifications=nexus3.edgexfoundry.org:10004/docker-support-notifications-go:master
-export supportScheduler=nexus3.edgexfoundry.org:10004/docker-support-scheduler-go:master
-export supportRulesengine=nexus3.edgexfoundry.org:10004/docker-support-rulesengine:master
-
-export deviceVirtual=nexus3.edgexfoundry.org:10004/docker-device-virtual-go:master
 export appService=nexus3.edgexfoundry.org:10004/docker-app-service-configurable:latest
```

Comment:
Also should be pulled from the compose file.

Reply:
The same reason as mentioned.